docs/finalproject/projects.md

```diff
@@ -4,7 +4,7 @@ title: Project Briefings
 sidebar_label: Project Briefings
 ---
 
-You can find here some examples of projects that you could chose to do or that have been done previously, to give you and your team inspiration.
+These are some examples of projects that could we chosen for your team or projects that have been done previously.
```

Comment:
This doesn't quite make sense as a sentence. It might be helpful to use Grammarly before committing. What about saying instead:

Here are some examples of previous final projects.

Reply:
But the first list of projects isn't once that has been done, they are possible briefings/projects. So this phrase wouldn't make sense, unless I remove all the possible briefings, which I am happy to do.

Reply:
I think all those projects have been done.

Reply:
That makes it easier, indeed. Will amend the PR now and if you can approve it in 5 minutes would be great. Thank you.